WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -649,7 +649,7 @@ public static void deletePostsBeforeGapMarkerForTag(ReaderTag tag) {
         }
 
         String dateColumn = getSortColumnForTag(tag);
-        String[] args = {gapMarkerDate, tag.getTagSlug(), Integer.toString(tag.tagType.toInt())};
+        String[] args = {tag.getTagSlug(), Integer.toString(tag.tagType.toInt()), gapMarkerDate};
```

Comment:
Wow, can't believe I missed this when I wrote the original code. Glad you caught it!

Reply:
👍